custom_button/main.py

```diff
@@ -0,0 +1,46 @@
+import tkinter as tk
+from tkinter import *
+from CustomButton import CustomButton
+
+def changing_information(button_list):
+    for x in button_list:
```

Comment:
Not `x`, always make appropriate names, I suggest here `button`.
I know that here it seems to be a totally unnecessary, but it's good to always and always trying make perfect and clear names.